src/common/components/registerAccountModal/registerAccountModal.tpl.html

```diff
@@ -36,7 +37,7 @@ <h3 translate>Create Your Account</h3>
     on-sign-in="$ctrl.onSignIn()"
     is-inside-another-modal="true"
   ></sign-up-modal>
-  <div ng-switch-when="loading">
+  <div ng-switch-when="loading-donor">
```

Comment:
You decided to keep the loading in the registerAccountModal?

Reply:
I wanted the "Checking your donor account" message to show up in the title. So I kept the register account modal loading state so that it can change the modal title.

A bunch of the modals inside `registerAccountModal` are setting `modalTitle`. But that variable doesn't seem to be used anywhere in the template, so it could use some refactoring.